eidas_node/connector/templates/eidas_node/connector/citizen_country_selector.html

```diff
@@ -0,0 +1,41 @@
+{% extends "eidas_node/connector/base.html" %}
```

Comment:
This is a duplicate if `country_selector.html`

Reply:
Removed.